skimage/measure/_regionprops.py

```diff
@@ -134,21 +156,24 @@ def euler_number(self):
         return -num + 1
 
     def extent(self):
-        rows, cols = self.image.shape
-        return self.moments[0, 0] / (rows * cols)
+        return float(self.area) / self.image.size
```

Comment:
I would prefer here that you use `from __future__ import division` (at top) and then just write `self.area / self.image.size`. This makes the code more Python 3-like, which is what we should aim for in general imho.